manifests/init.pp

```diff
@@ -117,5 +70,4 @@
   class { '::ntp::config': } ~>
   class { '::ntp::service': } ->
   anchor { 'ntp::end': }
```

Comment:
Without having it run, the solution might look like this:
```
 contain('ntp::config')
 contain('ntp::service')
 Class['ntp::config'] ~> Class['ntp::service']
```

I'm not sure I like that much better. I've created MODULES-3917 to follow up